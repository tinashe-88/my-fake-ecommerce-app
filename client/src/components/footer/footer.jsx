import React from 'react'

import {
    FooterContainer
} from './footer.styles'

export const Footer = () => (
  <>
    <FooterContainer>
      Copyright Â©{new Date().getFullYear()}.
      {` `}
      All rights reserved.
      {` `}
    </FooterContainer>
  </>
)

export default Footer