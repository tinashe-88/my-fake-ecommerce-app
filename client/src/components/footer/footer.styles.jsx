import styled from 'styled-components'

export const FooterContainer = styled.footer`
  padding-top: 5vh;
  min-height: 100px;
  text-align: center;
  font-size: .7rem;
  color: #777;
`