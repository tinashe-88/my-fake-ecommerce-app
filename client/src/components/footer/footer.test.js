import React from 'react'
import { shallow } from 'enzyme'

import { Footer } from './footer'

it('Should render Footer component', () => {
  expect(shallow(<Footer />)).toMatchSnapshot()
})