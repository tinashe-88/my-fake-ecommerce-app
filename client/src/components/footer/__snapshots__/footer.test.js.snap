// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer component 1`] = `
<Fragment>
  <styled.footer>
    Copyright Â©
    2020
    .
     
    All rights reserved.
     
  </styled.footer>
</Fragment>
`;
